static/documentation/8.1.0/b10/bower/datatables.net-buttons-bs/css/buttons.bootstrap.css

```
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```

Stylelint (17.13.0)

[error] Unknown word https (CssSyntaxError)

(CssSyntaxError)